content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -1,17 +1,35 @@
 # Use models
 
 Whether you need to download your models to use them, or you're looking to set
-up some automation in CI/CD to deploy them, Iterative Studio provides these
+up some automation in CI/CD to deploy them, DVC Studio provides these
 capabilities.
 
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using Studio [REST API], `dvc get`, or DVC [Python API].
+versions with DVC Studio from the web interface or in your code.
 
-### Using Studio API
+Prerequisites:
 
-Using Studio API will require you to add your remote storage secrets to
+- Model stored with DVC with S3 or Azure [remote].
+- The DVC Studio project you like to download your model from needs access to
+  your [remote storage credentials].
+- Access to your [DVC Studio Access token].
+
+Without these prerequisites, you can still [download a model artifact with DVC],
```

Comment:
Now it looks like this paragraph and the first one under `Using DVC Studio API` are repetitive. Can we combine them? Can we also drop the `Using DVC Studio API` header? It doesn't seem needed now.